### DISCUSSION TITLE
Embedding a form as a popup, and tracking which page a user was on when they submitted it?

### DISCUSSION THREAD

**Discussion by paul-oreilly-nz** [OpnForm/OpnForm#505] [Q&A]
It'd be great to know what part of my website a user was on when they submitted a popup form, so I can gain some additional insight into what they were reading or may be talking about.

I have made a hidden field for the URL of the website.

How would I best capture that information?

**Comment by JhumanJ** [comment]
Here's a guide :) https://help.opnform.com/en/article/how-to-track-the-source-of-your-opnform-submissions-auqbxs/
